src/components/homepage/Header.jsx

```diff
@@ -16,10 +17,10 @@ const Header=()=>{
             <ul className="nav-icons">
                 <li><Link to="/login" className="navbar-links" >Login </Link></li>
                 <div className="h-space-1rem"></div>
-                <li><Link to="" className="navbar-links" >
+                <li><Link to="/wishlist" className="navbar-links" >
                         <div className="badge-container">
                             <i className="fas fa-heart">
-                                <span className="badge-icon">0</span>
+                                <span className="badge-icon">{wishlist.length}</span>

```

Comment:
Add background colour/colour and border-radius here. so that rendered value will look more visible.